hub/util/visualization.py

```diff
@@ -0,0 +1,44 @@
+# type: ignore
+
+import numpy as np
+from hub.util.exceptions import (
+    ModuleNotInstalledException,
+    VisualizationError,
+)
+
+
+def _import_plt():
+    global plt
+    try:
+        import matplotlib.pyplot as plt
```

Comment:
Shall we add this into plugin dependancies? 

https://github.com/activeloopai/Hub/blob/main/requirements/plugins.txt